tasks/configure-Debian.yml

```diff
@@ -8,6 +8,22 @@
   with_items: "{{ apache_ports_configuration_items }}"
   notify: restart apache
 
+- name: Configure granularity of apache version displayed.
+  lineinfile:
+    dest: "{{ apache_server_root }}/{{ apache_daemon }}.conf"
+    regexp: "^ServerTokens "
+    line: "ServerTokens {{ apache_config_version_visibility }}"
+    state: present
+  notify: restart apache
+
+- name: Configure granularity of apache signature displayed.
+  lineinfile:
+    dest: "{{ apache_server_root }}/{{ apache_daemon }}.conf"
```

Comment:
On Debian 10 this setting is in `{{ apache_server_root }}/conf-available/security.conf` by default.